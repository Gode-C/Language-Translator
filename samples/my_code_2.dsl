/* Program that computes a number raised to specified power */

main () { 
  float base;
  base = 5;
  int exponent;
  exponent = 3;
  float result;
  result = 1;

  while(exponent != 0)
  {
  	result = result*base;
  	exponent = exponent - 1;
  }

  print("The answer is: ");
  print(result);
}

