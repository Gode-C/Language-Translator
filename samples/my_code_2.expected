The answer is: 625 
